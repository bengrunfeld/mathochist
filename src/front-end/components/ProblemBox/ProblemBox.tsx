import React from 'react'

const ProblemBox = () => (
  <div>
    <h1>ProblemBox</h1>
  </div>
)

export default ProblemBox
