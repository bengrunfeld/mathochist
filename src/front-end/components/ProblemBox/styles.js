import s from 'styled-components'

const ProblemBox = s.div`
`

const Caption = s.p`

`

export { ProblemBox, Caption }
