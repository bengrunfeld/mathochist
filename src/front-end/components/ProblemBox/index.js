import ProblemBox from './ProblemBox'

export default ProblemBox
