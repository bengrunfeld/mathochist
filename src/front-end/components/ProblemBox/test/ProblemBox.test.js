import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import ProblemBox from '../ProblemBox'

describe('<ProblemBox />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<ProblemBox />)
    expect(tree).toMatchSnapshot()
  })
})
