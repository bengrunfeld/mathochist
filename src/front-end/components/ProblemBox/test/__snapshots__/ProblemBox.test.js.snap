// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProblemBox /> matches the snapshot 1`] = `
<ProblemBox>
  <div>
    <h1>
      ProblemBox
    </h1>
  </div>
</ProblemBox>
`;
